src/config.ts

```diff
@@ -110,6 +128,9 @@ export function createProxyConfig(option: IProxyOption): IProxyConfig {
     const trustProxy =
         option.trustProxy || loadTrustProxy(process.env.TRUST_PROXY);
 
+    const parsedTags = option.tags || process.env.UNLEASH_TAGS;
+    const tagFilters = mapTagsToFilters(parsedTags);
+
```

Comment:
Looks good, but would be good to have a few tests inside config.test.ts, expecially for the mapTagsToFilter function.